packages/genesys-spark-components/src/components/beta/gux-announce/gux-announce.tsx

```diff
@@ -21,8 +21,8 @@ export class GuxAnnounce {
   @Prop()
   politeness: GuxAnnouncePoliteness = 'polite';
 
-  // eslint-disable-next-line @typescript-eslint/require-await
   @Method()
+  // eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
could not find out what changed to make these changes necessary but they are 🤷‍♂️

Reply:
Did something change in the linter to consider @Method() to be the next line? lol